ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -280,6 +280,10 @@ def test_function(
             **kwargs,
         )
 
+        assert ivy.nested_map(
+            ret_from_target, lambda x: ivy.is_ivy_array(x) if ivy.is_array(x) else True
```

Comment:
`ivy.is_ivy_array()` would be bound to global ivy, which has it's backend unchanged, we should use local Ivy.